src/shogun/evaluation/TimeSeriesSplitting.cpp

```diff
@@ -45,6 +45,7 @@ CTimeSeriesSplitting::CTimeSeriesSplitting(CLabels* labels, index_t num_subsets)
     : CSplittingStrategy(labels, num_subsets)
 {
 	init();
+	build_subsets();
```

Comment:
the method seem to be called in `init`, why call it again?

Reply:
build_subsets() you mean? It isn't called on init() in this class.